src/atoms/Text/Text.web.js

```diff
@@ -1,3 +1,139 @@
-const Text = () => null;
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import baseTheme from '../../tokens/theme';
+import isDefined from '../../_helpers/isDefined';
+import { getColorKeys, getColor, getLineHeight } from '../../_helpers/theme';
+
+/*
+ * Sets height for the div
+ * @param {Object} theme
+ * @param {String} size
+ * @param {number} maxLines
+ */
+const calculateHeight = ({ theme, size, maxLines }) => {
+  const lineHeight = getLineHeight(theme, size);
+  const containerHeight = `${parseFloat(lineHeight) * maxLines}px`;
+  return containerHeight;
+};
+
+const styles = {
+  fontFamily({ theme, _weight }) {
+    return theme.fonts.family.lato[_weight];
+  },
+  fontSize({ theme, size }) {
+    return theme.fonts.size[size];
+  },
+  color({ theme, color }) {
+    return getColor(theme, color);
+  },
+  align({ align }) {
+    switch (align) {
+      case 'left':
+      default:
+        return 'flex-start';
+      case 'center':
+        return 'center';
+      case 'right':
+        return 'flex-end';
+    }
+  },
+  letterSpacing({ theme, _letterSpacing }) {
+    return `${theme.fonts.letterSpacing[_letterSpacing]}`;
```

Comment:
when we don't have any condition then put this directly in the component styles itself below.

Reply:
based on our conversation , keeping it here since it is extracted via theme.

Reply:
theme is available inside styled props

Reply:
I have already pushed these changes, @kamleshchandnani